rdm/init_files/documents/test_record.md

```diff
@@ -40,6 +47,13 @@ It is ok if some tests do not trace to any particular requirements, however all
 {% endfor %}
 
 ## Integration Tests
+[[These are the results of automated integration testing 62304:5.6.3 62304:5.6.4 62304:9.8.a]]
+TODO: document any anomilies encountered [[62304:9.8.b]]
+TODO: document the software version tested [[62304:9.8.c]]
+TODO: document any relevant configuration [[62304:9.8.d]]
+TODO: document the relavent tools used to run these tests [[62304:9.8.e]]
+TODO: document the date tested [[62304:9.8.f]]
+TODO: document the identity of the tester if any manual steps were required[[62304:9.8.g]].
```

Comment:
One these TODOs are moved, per the previous comments, I think we can remove all of these TODOs.

Reply:
Fixed.